### ISSUE TITLE
No DAT support yet?

### ISSUE THREAD

**Issue by HTEredrock** [webrecorder/webrecorder#706]
Just checking to see if this is still in the works?: https://blog.webrecorder.io/2018/08/02/webrecorder-dat-integration.html

And why not make the whole site run on DAT?

**Comment by ikreymer** [comment]
Yes, it definitely is, we haven't had a chance to fully release this as part of the hosted webrecorder.io, but it is available for testing locally, if you're interested.

You can enable DAT by setting `ALLOW_DAT=1` in your `wr.env`, which will enable the Dat UI options.

We still need to add support for importing via Dat to the main application, it was only prototyped in the Webrecorder Player. We ran into some issues (I believe it was with loading from S3 sync) and haven't had time to fully production-ready. Having to balance multiple priorities has delayed this a bit, unfortunately.

But, if you'd interested in trying the feature out on webrecorder.io, and help test it, we can enable it on your account on webrecorder.io as well!

We are also working on a desktop version of Webrecorder and Dat support will be an important aspect of the tool.
